### PR TITLE
how-to/maintain: update slow log document

### PR DESCRIPTION
**Pull request by zz-jason**
# What is changed, added or deleted?  

1. refine some words in slow log document 
2. refine some sql and shell scripts in the document

### Which version does your change affect? 

v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: v3.0/how-to/maintain/identify-slow-queries.md [hunk 1]
```diff
@@ -6,170 +6,190 @@ aliases: ['/docs-cn/sql/slow-query/']
...
+* `Query`：表示 SQL 语句。慢日志里面不会打印 `Query`，但映射到内存表后，对应的字段叫 `Query`。
 * `Digest`：表示 SQL 语句的指纹。
-* `Memory_max`：表示执行期间 TiDB 使用的最大内存空间，单位为 byte。
-* `Query_time`：表示执行这个语句花费的时间。只有执行时间超过 slow-threshold 的语句才会输出这个日志，单位是秒，以下所有的时间字段的单位都是秒。
-* `Process_time`：执行 SQL 在 TiKV 的处理时间之和，因为数据会并行的发到 TiKV 执行，这个值可能会超过 `Query_time`。
-* `Wait_time`：表示这个语句在 TiKV 的等待时间之和，因为 TiKV 的 Coprocessor 线程数是有限的，当所有的 Coprocessor 线程都在工作的时候，请求会排队；当队列中有某些请求耗时很长的时候，后面的请求的等待时间都会增加。
+* `Txn_start_ts`：表示事务的开始时间戳，也是事务的唯一 ID，可以用这个值在 TiDB 日志中查找事务相关的其他日志。
+* `Is_internal`：表示是否为 TiDB 内部的 SQL 语句。`true` 表示 TiDB 系统内部执行的 SQL 语句，`false` 表示用户执行的 SQL 语句。
+* `Index_ids`：表示语句涉及到的索引的 ID。
+* `Succ`：表示语句是否执行成功。
```
Comment (dcalvin) [inline v3.0/how-to/maintain/identify-slow-queries.md]: `Succ` is really weird, and not necessarily associated with Success. We don't normally segment a word like this.   Can we use the full form (success or succeed)?

File: v3.0/how-to/maintain/identify-slow-queries.md [hunk 2]
```diff
@@ -183,26 +203,31 @@ tidb > select query, query_time, stats from INFORMATION_SCHEMA.`SLOW_QUERY` wher
...
+set tidb_slow_query_file = "/path-to-log/tidb-slow.log"
 ```
 
-`INFORMATION_SCHEMA.SLOW_QUERY` 目前暂时只支持解析一个慢日志文件，如果慢日志文件超过一定大小后被 logrotate 成多个文件了，查询 `INFORMATION_SCHEMA.SLOW_QUERY` 也只会去解析一个文件。后续我们会改善这点。
+## 用 `pt-query-digest` 工具分析 TiDB 慢日志
+
+可以用 `pt-query-digest` 工具分析 TiDB 慢日志。
 
-### 用 pt-query-digest 工具分析 TiDB 慢日志
+> 注：建议使用 pt-query-digest 3.0.13 及以上版本
```
Comment (dcalvin) [inline v3.0/how-to/maintain/identify-slow-queries.md]:
```suggestion
> **注意：**
>
> 建议使用 pt-query-digest 3.0.13 及以上版本。
```

File: v3.0/how-to/maintain/identify-slow-queries.md [hunk 3]
```diff
@@ -6,170 +6,190 @@ aliases: ['/docs-cn/sql/slow-query/']
...
-|            |   `Mem_max` bigint(20) unsigned DEFAULT NULL,               |
-|            |   `Query` varchar(4096) DEFAULT NULL                        |
-|            | ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
-+------------+-------------------------------------------------------------+
-```
-
-### 实现细节
-
-`INFORMATION_SCHEMA.SLOW_QUERY` 表里面的内容是通过实时解析 TiDB 慢日志里面的内容得到的。每次查询这个表时都会去读取慢日志文件里面的内容，然后解析。
+> 注：每次查询 `SLOW_QUERY` 表时，TiDB 都会去读取和解析一次当前的慢查询日志。
```
Comment (dcalvin) [inline v3.0/how-to/maintain/identify-slow-queries.md]:
```suggestion
> **注意：**
>
> 每次查询 `SLOW_QUERY` 表时，TiDB 都会去读取和解析一次当前的慢查询日志。
```

File: v3.0/how-to/maintain/identify-slow-queries.md (hunk 1 above)
Comment (zz-jason) [reply v3.0/how-to/maintain/identify-slow-queries.md]: Agreed. But it appears in the log file now, so I prefer to not change the field name. We can optimize it later in the `tidb` repo and update this document in the future.

File: dev/how-to/maintain/identify-slow-queries.md [hunk 4]
```diff
@@ -1,203 +1,198 @@
...
+
+* `User`：表示执行语句的用户名。
+* `Conn_ID`：表示用户的链接 ID，可以用类似 `con:3` 的关键字在 TiDB 日志中查找该链接相关的其他日志。
+* `DB`：表示执行语句时使用的 database。
+
+和 TiKV Coprocessor Task 相关的字段：
+
 * `Request_count`：表示这个语句发送的 Coprocessor 请求的数量。
 * `Total_keys`：表示 Coprocessor 扫过的 key 的数量。
+* `Process_time`：执行 SQL 在 TiKV 的处理时间之和，因为数据会并行的发到 TiKV 执行，这个值可能会超过 `Query_time`。
```
Comment (qw4990) [inline dev/how-to/maintain/identify-slow-queries.md]: Should we specify the time unit we are using for all fields like `XXX_time`?

Comment (qw4990) [reply dev/how-to/maintain/identify-slow-queries.md]: My mistake, the time unit is specified.

File: dev/how-to/maintain/identify-slow-queries.md [hunk 5]
```diff
@@ -1,203 +1,198 @@
...
 | query                       | query_time  |
 +-----------------------------+-------------+
 | select * from t1 where a=1; | 0.302558006 |
 | select * from t1 where a=2; | 0.401313532 |
 +-----------------------------+-------------+
-2 rows in set
 ```
 
-### 查询统计信息是 pseudo 的慢查询
+### 搜索统计信息为 pseudo 的慢查询 SQL
```
Comment (lilin90) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
### 搜索统计信息为 pseudo 的慢查询 SQL 语句
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 6]
```diff
@@ -1,163 +1,199 @@
...
 | query                       | query_time  |
 +-----------------------------+-------------+
 | select * from t1 where a=1; | 0.302558006 |
 | select * from t1 where a=2; | 0.401313532 |
 +-----------------------------+-------------+
-2 rows in set
 ```
 
-### 查询统计信息是 pseudo 的慢查询
+### 搜索统计信息为 pseudo 的慢查询 SQL
```
Comment (lilin90) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
### 搜索统计信息为 pseudo 的慢查询 SQL 语句
```

File: v3.0/how-to/maintain/identify-slow-queries.md [hunk 7]
```diff
@@ -6,170 +6,194 @@ aliases: ['/docs-cn/sql/slow-query/']
...
 | query                       | query_time  |
 +-----------------------------+-------------+
 | select * from t1 where a=1; | 0.302558006 |
 | select * from t1 where a=2; | 0.401313532 |
 +-----------------------------+-------------+
-2 rows in set
 ```
 
-### 查询统计信息是 pseudo 的慢查询
+### 搜索统计信息为 pseudo 的慢查询 SQL
```
Comment (lilin90) [inline v3.0/how-to/maintain/identify-slow-queries.md]:
```suggestion
### 搜索统计信息为 pseudo 的慢查询 SQL 语句
```

**Pull request merged**